### PR TITLE
feat: Add custom baseImage to Star

### PR DESCRIPTION
**Pull request by vinics12**
Possibility to add a image to be used as base at TapRating like swipe rating

### CODE REVIEW

File: src/components/Star.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ const STAR_SIZE = 40;
 
 export default class Star extends PureComponent {
   static defaultProps = {
+    baseStarImage: STAR_IMAGE,
```
Comment (sshivananda) [inline src/components/Star.js:10]: thoughts on naming this as `starImage` ?

Comment (vinics12) [reply src/components/Star.js]: updated 😉

**Comment by sshivananda** [comment]
Looks good - thanks for the PR !!

**Comment by sshivananda** [comment]
Released in v7.5.0

**Pull request merged**